src/model/cloud-runner/aws/aws-task-runner.ts

```diff
@@ -0,0 +1,216 @@
+import * as AWS from 'aws-sdk';
+import CloudRunnerEnvironmentVariable from '../services/cloud-runner-environment-variable';
+import * as core from '@actions/core';
+import CloudRunnerTaskDef from '../services/cloud-runner-task-def';
+import * as zlib from 'zlib';
+import CloudRunnerLogger from '../services/cloud-runner-logger';
+import { Input } from '../..';
+import fs from 'fs';
+import { CloudRunnerState } from '../state/cloud-runner-state';
+import { CloudRunnerStatics } from '../cloud-runner-statics';
+
+class AWSTaskRunner {
+  static async runTask(
+    taskDef: CloudRunnerTaskDef,
+    ECS: AWS.ECS,
+    CF: AWS.CloudFormation,
+    environment: CloudRunnerEnvironmentVariable[],
+    buildGuid: string,
+    commands: string[],
+  ) {
+    const cluster = taskDef.baseResources?.find((x) => x.LogicalResourceId === 'ECSCluster')?.PhysicalResourceId || '';
+    const taskDefinition =
+      taskDef.taskDefResources?.find((x) => x.LogicalResourceId === 'TaskDefinition')?.PhysicalResourceId || '';
+    const SubnetOne =
+      taskDef.baseResources?.find((x) => x.LogicalResourceId === 'PublicSubnetOne')?.PhysicalResourceId || '';
+    const SubnetTwo =
+      taskDef.baseResources?.find((x) => x.LogicalResourceId === 'PublicSubnetTwo')?.PhysicalResourceId || '';
+    const ContainerSecurityGroup =
+      taskDef.baseResources?.find((x) => x.LogicalResourceId === 'ContainerSecurityGroup')?.PhysicalResourceId || '';
+    const streamName =
+      taskDef.taskDefResources?.find((x) => x.LogicalResourceId === 'KinesisStream')?.PhysicalResourceId || '';
+
+    const task = await ECS.runTask({
+      cluster,
+      taskDefinition,
+      platformVersion: '1.4.0',
+      overrides: {
+        containerOverrides: [
+          {
+            name: taskDef.taskDefStackName,
+            environment: [...environment, { name: 'BUILDID', value: buildGuid }],
```

Comment:
Is this hardcoded `BUILDID` intended?

Reply:
I need to audit if this is still consumed anywhere. I believe it's used in the output build zip name.

But there will be a build id being serialized automatically too now, so will remove it.